examples/resnet-runtime.cpp

```diff
@@ -51,6 +51,14 @@ llvm::cl::opt<std::string> tracePath("trace-path",
                                      llvm::cl::desc("Write trace logs to disk"),
                                      llvm::cl::init(""),
                                      llvm::cl::cat(category));
+llvm::cl::opt<BackendKind> backend(
+    llvm::cl::desc("Backend to use:"), llvm::cl::Optional,
+    llvm::cl::values(clEnumValN(BackendKind::Interpreter, "interpreter",
+                                "Use interpreter (default option)"),
+                     clEnumValN(BackendKind::CPU, "cpu", "Use CPU"),
+                     clEnumValN(BackendKind::OpenCL, "opencl", "Use OpenCL"),
+                     clEnumValN(BackendKind::Habana, "Habana", "Use Habana")),
+    llvm::cl::init(BackendKind::Interpreter), llvm::cl::cat(category));
```

Comment:
nit: to preserve current semantics this needs to be cpu backend.

Reply:
Will change the default backendkind to cpu from interpreter.  Thank you.